src/position/test/position.spec.js

```diff
@@ -16,18 +16,26 @@ describe('position elements', function () {
     $position = _$position_;
   }));
   beforeEach(function () {
-    this.addMatchers({
-      toBePositionedAt: function(top, left) {
-        this.message = function() {
-          return 'Expected "('  + this.actual.top + ', ' + this.actual.left +  ')" to be positioned at (' + top + ', ' + left + ')';
+    jasmine.addMatchers({
+      toBePositionedAt: function(util, customEqualityTesters) {
+        return {
+          compare: function(actual, top, left) {
+            var result = {
+              pass: util.equals(actual.top, top, customEqualityTesters) &&
+                      util.equals(actual.left, left, customEqualityTesters)
```

Comment:
I think `util.equals` is better if it's comparison of an things that are of unknown types. It allows the user to specify custom equality testers and use them.
However, since we don't rely on any custom equality handling, I wonder if it's just a level of unnecessary indirection. @wesleycho any thoughts?